integration-tests/src/tests/client/process_blocks.rs

```diff
@@ -4511,3 +4511,162 @@ mod contract_precompilation_tests {
         assert!(caches[1].get(&contract_key.0).unwrap().is_none());
     }
 }
+
```

Comment:
Also it would be interesting to take a look on chunk cache code coverage running those tests.

Reply:
This is interesting. Do you know a tool for that?

Reply:
I guess this should help with that although I didn't try it out yet (particularly `A source-based code coverage`):
https://doc.rust-lang.org/nightly/rustc/instrument-coverage.html

(It's requires nightly rustc)